test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -33,114 +33,74 @@ static logger::Logger log_ = logger::testLog("BlockStore");
 
 class BlStore_Test : public ::testing::Test {
  protected:
+  void clear() {
+    try {
+      namespace fs = boost::filesystem;
+      using perms = boost::filesystem::perms;
+      fs::permissions(
+          block_store_path,
+          perms::owner_read | perms::owner_write | perms::owner_exe);
+      fs::remove_all(block_store_path);
+    } catch (...) {
```

Comment:
Why do you ignore exceptions?

Reply:
It is test -- it does its best to remove block_store_path and gives up in case of any trouble (permissions, no folder, etc).

Reply:
It is `setUp`, what if it fails and then tests start?

Reply:
This is okay. This test requires environment to be empty (to not have block_store_path dir). 

clear() is both in setup and teardown. 
It can fail only if
1. there is no folder block_store_path
2. no permissions to folder_block_store_path
3. maybe some other magical reason, like disk crash